src/main/java/org/opentripplanner/api/resource/GraphPathToTripPlanConverter.java

```diff
@@ -863,10 +863,6 @@ private static Place makePlace(State state, Vertex vertex, Edge edge, Stop stop,
             place.stopCode = stop.getCode();
             place.platformCode = stop.getCode();
             place.zoneId = stop.getZone();
-            if (endOfLeg) place.stopIndex++;
-            if (tripTimes != null) {
-                place.stopSequence = tripTimes.getStopSequence(place.stopIndex);
-            }
```

Comment:
The `endOfLeg` variable and `edge` parameter is no longer in use and should be removed.